.changeset/clean-frogs-dance.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+Updates aria-label support for consistency
```

Comment:
It seems like it would be useful to be a little more explicit and detailed here so consumers of the changelog understand they can now use `@ariaLabel` in more places than before?